docs/user/alerting/index.asciidoc

```diff
@@ -1,7 +1,7 @@
 include::alerting-getting-started.asciidoc[]
+include::create-and-manage-rules.asciidoc[]
 include::defining-rules.asciidoc[]
 include::rule-management.asciidoc[]
```

Comment:
Same comment as above: will remove links to `defining-rules` and `rule-management` when bad cross document links are resolved.